### DISCUSSION TITLE
Cannot run occ commands - Console has to be executed with the user that owns the file config/config.php

### DISCUSSION THREAD

**Discussion by arsaboo** [nextcloud/all-in-one#1503] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
I am trying to run the occ command `files:scan`. However, I keep getting this error:
```
arsaboo@nxtcloud:~$ sudo docker exec -it nextcloud-aio-nextcloud php occ files:scan --all
Console has to be executed with the user that owns the file config/config.php
Current user id: 0
Owner id of config.php: 33
Try adding 'sudo -u #33' to the beginning of the command (without the single quotes)
If running with 'docker exec' try adding the option '-u 33' to the docker command (without the single quotes)
```

### Expected behavior 
The occ command should run and scan all the files
### Actual behavior 
Keep getting the above error message

### Host OS 
Docker on Ubuntu 

#### Nextcloud AIO version 
3.1.0
#### Current channel 
Latest

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#how-to-run-occ-commands.

Based on that should the correct command be 
`sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ files:scan --all`

**Reply by arsaboo** [reply]
That works...thanks a lot!

**Comment by BeachBum727** [comment]
Same issue.
Nextcloud AIO installation on unRaid via docker.
The above command worked for me when executed via unRaid console. Not the Nextcloud console.

Thanks !!
